src/main/scala/chiseltest/backends/TreadleBackend.scala

```diff
@@ -0,0 +1,17 @@
+package chiseltest.backends
+
+import firrtl.AnnotationSeq
+import treadle.{TreadleCircuitStateAnnotation, TreadleTester}
+import treadle.stage.phases.PrepareAst
+
+object TreadleBackend extends SimulatorBackend {
+  def compileDut(annotations: AnnotationSeq) = {
+    val treadleAnnotations = (new PrepareAst).transform(annotations)
+    (treadleAnnotations :+ SimulatorInterfaceAnnotation(
+      new TreadleInterface(new TreadleTester(treadleAnnotations))
+    )).filterNot {
+      case _: TreadleCircuitStateAnnotation => true
```

Comment:
Use a collect instead?

Reply:
will fix.

Reply:
I think here should be `filterNot` to remove `TreadleCircuitStateAnnotation` only.